tox.ini

```diff
@@ -10,7 +10,6 @@ envlist = py27, flake8
 addopts = --cov=mozetl
 
 [testenv]
-usedevelop = True
```

Comment:
This was the issue that was causing the error with the dependencies. I don't think `python install -e` likes the name for `pyspark==2.2.0.post0`.